addons/portal_rating/static/src/chatter/frontend/composer_patch.js

```diff
@@ -26,12 +26,15 @@ patch(Composer.prototype, {
     },
 
     async savePublisherComment() {
-        const data = await rpc("/website/rating/comment", {
-            rating_id: this.message.rating.id,
-            publisher_comment: this.props.composer.text.trim(),
-        });
-        this.message.rating = data;
-        this.props.onPostCallback();
+        if (this.state.active) {
+            this.state.active = false;
+            const data = await rpc("/website/rating/comment", {
+                rating_id: this.message.rating.id,
+                publisher_comment: this.props.composer.text.trim(),
+            });
+            this.message.rating = data;
+            this.props.onPostCallback();
+        }
```

Comment:
Can early return, as to not add an indent level to all of this, and minimise the diff